src/index.html

```diff
@@ -1,12 +1,174 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <link rel="stylesheet" type="text/css" href="Styles.css"/>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://use.fontawesome.com/releases/v5.7.1/css/all.css" rel="stylesheet"/>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <meta charset="UTF-8">
     <title>Whats Cookin</title>
   </head>
   <body>
-    <h1>Whats Cookin</h1>
+    <header class="nav-bar">
+      <div>
+        <h1>What's Cookin?</h1>
+      </div>
+      <div class="nav-btn-order">
+        <button class="nav-btn recipe-btn"><span>Recipes </span></button>
+        <button class="nav-btn fav-recipe-btn"><span>Favorite Recipes </span></button>
+        <button class="nav-btn pantry-btn"><span>Pantry </span></button>
+      </div>
+    </header>
+    <div class="solid-green"></div>
+    <section class="center-content">
+      <!-- <i class="fas fa-search"></i> -->
+      <input type="text" placeholder="Search Recipes" id="search-bar"/>
+    </section>
+    <hr />
+    <main class="main-order-row">
+      <section class="center-column side-tag-display">
+        <h1>Tags</h1>
+        <article class="tag-list">
+          <input type="checkbox" id="item1" />
+          <label for="item1">breakfast</label><br />
+          <input type="checkbox" id="item1" />
+          <label for="item1">lunch</label><br />
+          <input type="checkbox" id="item1" />
+          <label for="item1">dinner</label><br />
+          <input type="checkbox" id="item1" />
+          <label for="item1">dessert</label><br />
+          <input type="checkbox" id="item1" />
+          <label for="item1">drink</label><br />
+        </article>
+      </section>
+      <section class="recipe-cards all-recipes">
+        <section class="recipe-card-display center-column">
+          <i class="far fa-bookmark fa-4x"></i>
+          <img src="https://spoonacular.com/recipeImages/595736-556x370.jpg"/>
+          <h4>Chocolate Chip Cookies</h4>
+          <div class="tag-box">
+            <p class="tag">dessert</p>
+            <p class="tag">snack</p>
+          </div>
+        </section>
```

Comment:
Basic structure of the recipe card looks good and agreeable, depending on if we use micromodal or not we may need to expand this to format ingredients, directions, and etc.